Hazelcast.Net/Hazelcast.Client.Spi/ClientListenerService.cs

```diff
@@ -84,7 +84,7 @@ public ClientListenerService(HazelcastClient client)
         public string RegisterListener(IClientMessage registrationMessage, DecodeRegistrationResponse responseDecoder,
             EncodeDeregisterListenerRequest encodeDeregisterListenerRequest, DistributedEventHandler eventHandler)
         {
-            Debug.Assert(!Thread.CurrentThread.Name.Contains("eventRegistration"));
+            Debug.Assert(Thread.CurrentThread.Name == null || !Thread.CurrentThread.Name.Contains("eventRegistration"));
```

Comment:
Why did we need this change ? It does not look safe.

Reply:
Default thread name in .net is null. We want to assert that the thread is not eventRegistration thread. But the thread can be a user thread which its name by default null.